lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -50,6 +50,10 @@ export class AjvSchemaValidator implements SchemaValidator {
       ...options,
     });
     ajvEnableAll(this.ajv, loader);
+
+    if (options && (options as any).isArmCall === true) {
```

Comment:
Why options as any?

Reply:
Consider add interface SchemaValidatorOptions with isArmCall, then LiveValidatorOptions would inherit SchemaValidatorOptions

Reply:
LiveValidatorLoaderOptions cannot transform to ajv.Options.

Reply:
You need another option not for ajv.Options